pkg/scraper/decode.go

```diff
@@ -57,45 +54,48 @@ func decodeBatch(summary *stats.Summary) (*storage.MetricsBatch, error) {
 		num++
 	}
 	res.Pods = res.Pods[:num]
-	return res, utilerrors.NewAggregate(errs)
+	return res
 }
 
-func decodeNodeStats(nodeStats *stats.NodeStats, target *storage.NodeMetricsPoint) []error {
+func decodeNodeStats(nodeStats *stats.NodeStats, target *storage.NodeMetricsPoint) (success bool) {
 	timestamp, err := getScrapeTime(nodeStats.CPU, nodeStats.Memory)
 	if err != nil {
 		// if we can't get a timestamp, assume bad data in general
-		return []error{fmt.Errorf("unable to get valid timestamp for metric point for node %q, discarding data: %v", nodeStats.NodeName, err)}
+		klog.V(1).Infof("Skip metric for node %q, error: %v", nodeStats.NodeName, err)
```

Comment:
this may sound a bit paranoid, but can we track this internally as a Prometheus metrics counter and expose those failures at least as metrics?

Reply:
I think that's a good idea for metric. Do you think we can address it in separate Issue?

Reply:
sounds good to me, but I suggest implementing it before another release though.